src/appleseed/foundation/image/exrimagefilewriter.h

```diff
@@ -62,6 +62,12 @@ class APPLESEED_DLLSYMBOL EXRImageFileWriter
   : public IImageFileWriter
 {
   public:
+    // Constructor.
+    EXRImageFileWriter();
+
+    // Destructor.
+    ~EXRImageFileWriter() override;
```

Comment:
clang-tidy removes virtual when adding overrides, which I think it makes sense.
I wanted to be consistent with that in preparation of start running clang-tidy on appleseed's code base.